modules/adpod.js

```diff
@@ -457,7 +467,27 @@ export function getTargeting({codes, callback} = {}) {
 
   let bids = getBidsForAdpod(bidsReceived, adPodAdUnits);
   bids = (competiveExclusionEnabled || deferCachingEnabled) ? getExclusiveBids(bids) : bids;
-  bids.sort(sortByPricePerSecond);
+
+  let prioritizeDeals = config.getConfig('adpod.prioritizeDeals');
+  if (prioritizeDeals) {
+    let [otherBids, highPriorityDealBids] = bids.reduce((partitions, bid) => {
+      let bidDealTier = utils.deepAccess(bid, 'video.dealTier');
+      let minDealTier = config.getConfig(`adpod.dealTier.${bid.bidderCode}.minDealTier`);
+      if (minDealTier && bidDealTier) {
+        partitions[Number(bidDealTier > minDealTier)].push(bid);
+      } else if (bidDealTier) {
+        partitions[1].push(bid)
+      } else {
+        partitions[0].push(bid);
+      }
+      return partitions;
+    }, [[], []]);
+    highPriorityDealBids.sort(sortByPricePerSecond);
```

Comment:
To confirm - we're sorting these deal bids by the CPM?  Is the deal tier value synonymous with CPM overall speaking?  If not - should the sorting be done by the number in the deal tier rather than CPM (if I'm understanding this overall deal tier correctly)?

Reply:
We are always sorting on price per second for deal bids. Appnexus deal tier is from 0 to 20. Consider a bidder whose deal tier vary from 0 to 100 may get unfair advantage if i sort on deal tier. 
Also I do not have any other use case due to limited adoption of adpod.